internal/service/route53/record.go

```diff
@@ -652,16 +652,13 @@ func findRecord(d *schema.ResourceData, meta interface{}) (*route53.ResourceReco
 	// If this isn't a Weighted, Latency, Geo, or Failover resource with
 	// a SetIdentifier we only need to look at the first record in the response since there can be
 	// only one
-	maxItems := "1"
+	//maxItems := "1"
```

Comment:
```suggestion
	// maxItems := "1"
```

В настройках goland:  Editor > Code Style > Go > Other > Add leading space to comments